packages/k6/src/config/index.ts

```diff
@@ -12,15 +12,16 @@ export type AdapterNames =
   | 'coinmarketcap'
 
 interface AdapterConfig {
-  name: AdapterNames
+  name: AdapterNames | string
   /**
    * How many seconds to wait inbetween each call to this adapter
    */
   secondsPerCall: number
 }
 export const ADAPTERS: AdapterConfig[] = [
   // { name: 'nomics', secondsPerCall: 10 },
-  { name: 'coinapi', secondsPerCall: 10 },
+  // { name: 'coinapi', secondsPerCall: 10 },
   // { name: 'cryptocompare', secondsPerCall: 1 },
   // { name: 'tiingo', secondsPerCall: 5 },
+  { name: 'coingecko', secondsPerCall: 10 },
```

Comment:
Do these serve a purpose? Just wondering if we're supposed to add/comment out in this list when we want to do tests

Reply:
These are in here for how you could run the test before I made changes. You can still use the test in this way by adding/commenting out in the list like you could before. I probably should remove my change on this though as it is not necessary for this changeset.